uber/models/attendee.py

```diff
@@ -558,7 +642,7 @@ def badge(self):
     @property
     def is_transferable(self):
         return not self.is_new and \
-            not self.trusted_somewhere and \
+            not self.has_role_somewhere and \
```

Comment:
This is tricky. I think it's fine for now, but roles are not necessarily trusted roles, so we'll want to keep an eye on if we should remove this limitation for transferring badges.

Reply:
Yeah, for now it looks like we'll basically be treating any role as similar to being marked as "trusted" in the old system, which seems like a good way to do it for now.  I'm sure that roles will eventually evolve over time as people start to actually use them and submit feature requests.

Reply:
Yeah, this is just to keep the system functionally the same. We'll probably want to rethink it later.